### ISSUE TITLE
Define an option if we have just one slide

### ISSUE THREAD

**Issue by flayks** [nicinabox/superslides#4]
Hello!

I tried yesterday to make a script detecting if the slider has 1 or more slides to run some or some function... I loss several hours  to get absolutely nothing... and I finally found this variable in your script code :

``` javascript
size = $children.length;
```

**Why not create another parameter to set if yes or not we run the slider when we have just one slide ?!?**

Juste like that.

``` javascript
    start = function() {
      if (size > 1){ // Check if more than 1 slide
         animate((first_load ? 0 : "next"));
         return play();
      }
    };
```

It works very well and it offers the possibility of an adaptive slider, so, why don't implement it ? :)

**Comment by flayks** [comment]
That's maybe not the solution, but think about this possibility of just one slide on the slider. Without animation or negative left position, etc. It will be good for ajax call (like my case) !

**Comment by nicinabox** [comment]
Ah yes, it should be a little more graceful when only a single slide exists. I'll mark it as a bug for now and try to look at it this week.

**Comment by nicinabox** [comment]
I think this should work. Try out the file from the dev branch. https://github.com/nicinabox/superslides/blob/d30799756316f40f30b42c8e38c28b58635db823/build/jquery.superslides.js

**Comment by flayks** [comment]
Works like a charm. Thank you very much for this adding :)

**Comment by flayks** [comment]
Huh, it works, but I think the "resize" image function to the browser is not applied for the only one image on the slide.
I mean, the image is not translated with negative left in css. Am I right ?

**Comment by nicinabox** [comment]
Ah crap yeah. I didn't catch it on my small screen

**Comment by nicinabox** [comment]
Okay, try the most recent on the dev branch now. Should work.

**Comment by flayks** [comment]
Nop!

The only stretching is from the css (it doesn't center the image horizontally)

``` css
#bg li img{position:absolute; max-width:none; min-width:100%; min-height:100%; -ms-interpolation-mode:bicubic; -webkit-backface-visibility:hidden;}
```

**Comment by nicinabox** [comment]
Ah ha! It was a (separate) issue with image size calculation. It should now center in all directions correctly.

**Comment by flayks** [comment]
Yay! Finally, it works :)

You rock!

**Comment by nicinabox** [comment]
Thanks for your diligence in checking my work :)

**Comment by randalldon** [comment]
I was just wondering if the fix for a single slide has been included in version 0.4.2. Currently, if i only have a single slide this is completely hidden. The content is output to html but visually i can't see anything.
